lib/src/arithmetic/values/fixed_point_value.dart

```diff
@@ -143,18 +143,58 @@ class FixedPointValue implements Comparable<FixedPointValue> {
     return compareTo(other) == 0;
   }
 
-  /// Constructs [FixedPointValue] of a Dart [double] rounding away from zero.
+  /// Return a string representation of FloatingPointValue.
+  ///  return sign, exponent, mantissa as binary strings.
+  @override
+  String toString() =>
+      "(${signed ? '${value[-1].toString(includeWidth: false)} ' : ''}"
+      "${(m > 0) ? '${value.slice(m + n - 1, n).bitString} ' : ''}"
+      '${value.slice(n - 1, 0).toString(includeWidth: false)})';
+
+  /// Return true if double [val] be stored in FixedPointValue with [m] and [n]
```

Comment:
typo: "val be stored"

also, does this mean "without precision loss"?

Reply:
overflow.  There will be precision loss.  I'll update.
Unfortunately, fixed-point has no Infinity support.